.github/workflows/ci.yml

```diff
@@ -45,21 +45,21 @@ jobs:
 
       # Check all RISC-V targets:
       - name: check (esp32c3)
-        run: cd esp-hal-smartled/ && cargo +nightly check --features=esp32c3
+        run: cd esp-hal-smartled/ && cargo +nightly check --features=esp32c3 --target=riscv32imc-unknown-none-elf
```

Comment:
Why does the target matter here? `cargo check` does not perform code generation, it should not make any difference should it?

Reply:
See below